tests/kafkatest/tests/streams/streams_broker_compatibility_test.py

```diff
@@ -66,14 +67,11 @@ def test_compatible_brokers(self, broker_version):
 
         self.processor.wait()
 
-        num_consumed_mgs = self.consumer.total_consumed()
+        wait_until(lambda: self.consumer.total_consumed() == 1, timeout_sec=30, err_msg="Did expect to read exactly one message but got none within 30 seconds")
```

Comment:
Well. The Streams app writes exactly one message -- thus, it seems to be fair to expect exactly one. No?

I am wondering, how it could happen that it fails with more than one? It should either be zero if something went wrong or one.